### DISCUSSION TITLE
Regex validated input field

### DISCUSSION THREAD

**Discussion by gauravkrp** [directus/directus#11731] [Feature Requests]
When creating a new string type field for a collection, there should be an option to validate this input against a regex pattern provided at the time of field creation. This will drastically enhance the CMS feature to accommodate the general use-cases to correctly store data like `mobile-numbers`, `email`, `URL`, `pincodes`, or any other regex pattern that one could think of.

Attached image shows my suggestion for this feature request - 
![image](https://user-images.githubusercontent.com/12250556/154793933-fc95b65b-f3fb-4983-bcfd-22d41866a270.png)

**Comment by benhaynes** [comment]
@azrikahar any updates on field level validation? Will this be included?

**Reply by rijkvanzanten** [reply]
@benhaynes We already have a `_regex` validation operator, so once we have field-level validation configuration, we'll also have regex support 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
